plugins/destination/bigquery/client/migrate.go

```diff
@@ -24,7 +24,7 @@ func (c *Client) MigrateTables(ctx context.Context, msgs message.WriteMigrateTab
 	eg.SetLimit(concurrentMigrations)
 	for _, msg := range msgs {
 		if len(msg.Table.PrimaryKeys()) > 0 {
-			return fmt.Errorf("primary keys are not supported by the BigQuery plugin (if you are trying to use it as a state backend, this is not currently supported)")
+			return fmt.Errorf("primary keys are not supported by the BigQuery plugin. Hint: try setting `write_mode: append` in the destination spec")
```

Comment:
It's a shame we can't validate the write_mode in the plugin anymore 🤔

Reply:
Yeah, I think the CLI should do it and the plugin report the supported modes